src/Mount.cpp

```diff
@@ -2740,7 +2767,7 @@ void Mount::calculateStepperPositions(float raCoord, float decCoord, long& raPos
 //
 // This code tells the steppers to what location to move to, given the select right ascension and declination
 /////////////////////////////////
-void Mount::calculateRAandDECSteppers(long& targetRASteps, long& targetDECSteps) const {
+void Mount::calculateRAandDECSteppers(long& targetRASteps, long& targetDECSteps, long* pSolutions) const {
```

Comment:
I would do `long pSolutions[6]` to get compiler array length-checking

Reply:
Ohh, didn't even know that's legal syntax. Fixed.